native/android/.circleci

```diff
@@ -0,0 +1 @@
+../.circleci
```

Comment:
Why do we need this here as well?

Reply:
I thought I'd add this here as you can have the integreat-app-android project open and nothing else. I kind of added it for consistency. But I agree that we could remove it.